Corona-Warn-App/src/test/java/de/rki/coronawarnapp/bugreporting/censors/RatQrCodeCensorTest.kt

```diff
@@ -56,11 +56,6 @@ internal class RatQrCodeCensorTest {
         censor.checkLog(logLineToCensor) shouldBe logLineToCensor.copy(
             message = "Here comes the hash: SHA256HASH-ENDING-WITH-15ad of the rat test of RATest/FirstName RATest/LastName. He was born on RATest/DateOfBirth"
         )
-
-        every { CWADebug.isDeviceForTestersBuild } returns true
```

Comment:
Can be removed here too: 

```
mockkObject(CWADebug)
every { CWADebug.isDeviceForTestersBuild } returns false
```

Reply:
Ahhh i thought i got all of them 😄 should be gone now 👍